### PR TITLE
fix armcc license fail

### PR DESCRIPTION
**Pull request by yuhaoth**
Add more license server to fix that

https://jenkins-mbedtls.oss.arm.com/blue/organizations/jenkins/mbed-tls-pr-merge/detail/PR-5679-merge/33/pipeline/393 report arm license fail. 
And all build report same error now.

### CODE REVIEW

File: resources/docker_files/ubuntu-20.04/Dockerfile [hunk 1]
```diff
@@ -44,7 +44,7 @@ RUN apt-get update -q && apt-get install -yq \
         # for Mbed TLS tests
         abi-compliance-checker \
         # Note that there is a known issue #5332 that stock abi tools
-        # in ubuntu20.04 do not fail as expected.
```
Comment (daverodgman) [inline resources/docker_files/ubuntu-20.04/Dockerfile:47]: Typo (but not worth fixing unless you need to rework the PR for some other reason)

**Review by daverodgman — DISMISSED** [review]
Tested all the new addresses manually using e.g.:

./lmutil lmstat -c 7010@euhpc-lic03.euhpc.arm.com

The majority worked fine and are running a suitable license server, although the Bangalore ones all seemed to be down. I don't have armcc handy so wasn't able to check that they provide a license for the Arm tools.

LGTM (but as @gilles-peskine-arm says, it would be sensible to test this properly before merging).

**Comment by yuhaoth** [comment]
I test all server with below script. 
And got active license server list.`ARMLMD_LICENSE_FILE=7010@euhpc-lic03.euhpc.arm.com:7010@euhpc-lic04.euhpc.arm.com:7010@euhpc-lic07.euhpc.arm.com:7010@euhpc-lic06.euhpc.arm.com:7010@nahpc-lic06.nahpc.arm.com:7010@nahpc-lic09.nahpc.arm.com`
Is it worth to update to current license servers list ? @gilles-peskine-arm  and @daverodgman 

And original license servers are euhpc-lic03,euhpc-lic04,euhpc-lic05,euhpc-lic07. 


```python
import socket
import subprocess
from multiprocessing.dummy import  Pool

def host_names():
    for i in range(10):
        yield 7010, f'euhpc-lic{i:02d}.euhpc.arm.com'
        yield 8224, f'euhpc-lic{i:02d}.euhpc.arm.com'
    for i in range(10):
        yield 7010, f'nahpc-lic{i:02d}.nahpc.arm.com'
        yield 8224, f'nahpc-lic{i:02d}.nahpc.arm.com'
    for i in range(10):
        yield 7010, f'blr-lic{i:02d}.blr.arm.com'
        yield 8224, f'blr-lic{i:02d}.blr.arm.com'
    for i in range(10):
        yield 224, f'blr-lic{i:02d}.blr.arm.com'
def test_body(a):
    port, host = a
    try:

        subprocess.check_output(f'ARMLMD_LICENSE_FILE={port}@{host} /usr/local/ARM_Compiler_5.06u3/bin/armcc -c a.c', stderr=subprocess.STDOUT, shell=True)
        return port,host,socket.gethostbyname_ex(host)[2][0]
    except socket.gaierror:
        pass
    except :
        pass
servers=[]
with Pool() as pool:
    for server in pool.imap_unordered(test_body, host_names()):
        if not server:
            continue
        port, host, addr = server
        servers.append(f'{port}@{host}')
print(f'ARMLMD_LICENSE_FILE={":".join(servers)}')
```

**Comment by daverodgman** [comment]
I would try to find out what the recommend list of license servers is. Someone in DSG should know (have a look on Confluence). If you can't find someone who can give you a canonical list, the existing PR looks OK to me.

**Comment by yuhaoth** [comment]
> I would try to find out what the recommend list of license servers is. Someone in DSG should know (have a look on Confluence). If you can't find someone who can give you a canonical list, the existing PR looks OK to me.


> I would try to find out what the recommend list of license servers is. Someone in DSG should know (have a look on Confluence). If you can't find someone who can give you a canonical list, the existing PR looks OK to me.

Just update it base on DSG confluence page. And replace origin license servers with DNS name.

**Review by paul-elliott-arm — CHANGES_REQUESTED** [review]
This will work for the internal CI (because I believe it is iat least partially nside the Arm Network), but it won't work for the OpenCI, as euhpc.arm.com and ap01.arm.com are not resolvable outside of the Arm Network, hence our use of IP addresses prior to this.

**Comment by daverodgman** [comment]
> This will work for the internal CI (because I believe it is iat least partially nside the Arm Network), but it won't work for the OpenCI,

In that case I recommend closing this PR, since we are moving to OpenCI in the near future.

**Comment by yuhaoth** [comment]
According to Dave's comments, close it

**Pull request closed without merging**